packages/ckeditor5-table/docs/_snippets/features/table.js

```diff
@@ -21,16 +21,6 @@ ClassicEditor
 	} )
 	.then( editor => {
 		window.editor = editor;
-
-		window.attachTourBalloon( {
-			target: window.findToolbarItem( editor.ui.view.toolbar,
-				item => item.buttonView && item.buttonView.label && item.buttonView.label === 'Insert table' ),
-			text: 'Click to create a table.',
-			editor,
-			tippyOptions: {
-				placement: 'bottom-start'
-			}
-		} );
```

Comment:
Why are you removing this?

Reply:
A tour balloon in a current form can be attached only to elements that exist. A table toolbar is created when the table is focused. We would have to improve the tour balloon function to attach balloons more dynamically.

Reply:
But this is not the caption snippet, it's the generic table snippet.

Reply:
My bad then. Reverting.